cityofaustin/joplin
# 4184 event page janis urls

ZenHub Issue: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/4184

Deployed: https://joplin-pr-4184-event-urls.herokuapp.com/

API Query to check guide page url: https://joplin-pr-4184-event-urls.herokuapp.com/api/graphiql#query=%7B%0A%20%20allEventPages%20%7B%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20janisUrls%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D

# Description

Events should have urls now. Only when dates are added, however. 

I've  also added a couple tests as well as.... 
....went out of scope to add some query stuff to locations. 

# How can this be tested?
- Create an event
- See the allevents graphiql link above. 
- Events should NOT have any urls yet. Should be an empty array.
- Now add a *data* and a *location* to your event. 
- Now, the query should show a janis url. 

# Types of changes


# Checklist:
